### PR TITLE
Assert that tree exists, fix tests

### PR DESCRIPTION
**Pull request by timwis**

### CODE REVIEW

File: tests/browser/routing.js [hunk 1]
```diff
@@ -152,30 +156,14 @@ test('routing', function (t) {
...
+          t.deepEqual(state.location.params, {user: '123'})
+          return document.createElement('div')
         }]
       ]]
     ])
 
     app.start()
   })
-
-  t.test('prev.params always exists', function (t) {
```
Comment (yoshuawuyts) [inline tests/browser/routing.js]: lol yeah fair enough (:

File: index.js [hunk 2]
```diff
@@ -64,6 +64,7 @@ function choo (opts) {
     const state = _store.state({state: {}})
 
     const tree = _router(state.location.href, state)
+    assert.ok(tree, 'tree should exist')
```
Comment (yoshuawuyts) [inline index.js]: could you change this to `choo.start: the router should always return a valid DOM node`; perhaps even do a `t.equal(typeof tree, 'object')` also :sparkles:

**Comment by timwis** [comment]
@yoshuawuyts added the more helpful messages you requested

**Comment by timwis** [comment]
Hey @yoshuawuyts are we okay to merge this? Or are tests no longer failing?

**Comment by yoshuawuyts** [comment]
Go ahead!✨ - could you also run a release for this? Thanks!

**Pull request merged**